integr8ly/installation
# Update Enmasse version to beta

## Motivation
Update Enmasse version to AMQ Online beta

## What
Allow installation of the beta version of AMQ Online and set that version as the default version to be installed.

## Progress
- [x] Isolate all enmasse changes from this PR https://github.com/integr8ly/installation/pull/245
- [x] Patch the `keycloak-controller` to use the image `docker.io/lulf/amqmaas10-auth-controller-openshift:1.0-2`
- [x] Patch the `keycloak-plugin` init container in the `keycloak` deployment to use the image `docker.io/lulf/amqmaas10-auth-plugin-openshift:1.0-2`


## Verification Steps
1. Run the installer
2. Ensure that AMQ Online is successfully provisioned
3. Ensure that the pods deployed are using the correct images.
    - **address-space-controller**: `rh-amqmaas-1-tech-preview/amqmaas10-address-space-controller-openshift:1.0`
    - **api-server**: `rh-amqmaas-1-tech-preview/amqmaas10-api-server-openshift:1.0`
    - **keycloak-controller**: `docker.io/lulf/amqmaas10-auth-controller-openshift:1.0-2`
    - **service-broker**:`rh-amqmaas-1-tech-preview/amqmaas10-service-broker-openshift:1.0`
    - **keycloak-plugin**: `docker.io/lulf/amqmaas10-auth-plugin-openshift:1.0-2`

## Checklist:
